libs/core/src/lib/micro-process-flow/components/micro-process-flow/micro-process-flow.component.scss

```diff
@@ -0,0 +1,214 @@
+@import "../../mixins.scss";
+@import "../../variables.scss";
+
+// TODO: Move all styles to fundamental-styles library
+
+.#{$block} {
+    @include fd-reset();
+    @include fd-flex();
+
+    max-width: $max-container-width;
+    overflow: hidden;
+
+    &--independent-steps {
+        .#{$block}__connector {
+            &:before {
+                border-color: transparent;
+            }
+        }
+    }
+}
+
+.#{$block}__wrapper {
+    @include fd-reset();
+    @include fd-flex();
+
+    //transition: transform .3s ease;
+}
+
+.#{$block}__overflow-container {
```

Comment:
you will need to modify this when you move the code to fund-styles. We usually do it with nesting:
```
.#{$block} {
    &__overflow-container {...}
    &__wrapper {...}
    ....
}
```

but it's ok to leave it like this here if you did it for readability purposes